### ISSUE TITLE
[Feature] Add Nostr badge on websites list

### ISSUE THREAD

**Issue by bitcoinuser** [getAlby/lightning-browser-extension#1784]
### Feature description

I would like to see Nostr badge for websites I connected using Alby Nostr.

### Describe the solution

Same thing like this https://github.com/getAlby/lightning-browser-extension/pull/1779 but for Nostr

### Describe alternatives

No alternative.

### Are you working on this?

No

**Comment by manavdesai27** [comment]
I would like to work on this issue. Could you assign me?

**Comment by manavdesai27** [comment]
I guess we need to add `nostrAuth` to check if we are connected using nostr, as done in `lnurlauth`

**Comment by bumi** [comment]
awesome if you want to look into it. 
we have recently introduced the permissions. Those permissions are similar to the `lnurlAuth` flag. If a website has a permission it can execute certain calls. 

So I think it would be best to list the permission names as badges - and then maybe later also move `lnurlAuth` from a flag into a permission. 

I am not super sure how that would look like but I think this is the direction we should investigate here. 

(note: permissions are website AND account specific now)

**Comment by manavdesai27** [comment]
Okay @bumi let me look there.

**Comment by manavdesai27** [comment]
![image](https://user-images.githubusercontent.com/74018438/217921958-c5dcd651-a0eb-406d-a5da-3ed1a904a536.png)

Why is permission array empty for nostrich when I have logged in using the extension? @bumi

**Comment by manavdesai27** [comment]
I could check if the website has any permissions in the permission array, but how would I know if it is logged in using nostr?

**Comment by bumi** [comment]
we would just list the permissions as badges - and don't focus on the "login".

**Comment by manavdesai27** [comment]
Okay, what if there are more than one permissions? A horizontal list of badges?

**Comment by bumi** [comment]
I think so, at least currently - though the list could get long.
maybe we can include @jakubszwedo here, do you have ideas?

**Comment by manavdesai27** [comment]
Okayy. Let me see how to implement this.

**Comment by manavdesai27** [comment]
@bumi How do I add labels in translation for different permissions?

**Comment by manavdesai27** [comment]
![image](https://user-images.githubusercontent.com/74018438/221349669-43f2cd6c-4727-4861-af40-d646958e646c.png)
Currently this is happening.

**Comment by bumi** [comment]
@reneaaron @reneaaron what do we do here best? I guess those should be new translations, correct? 

@manavdesai27 it looks good, but you need to do: 

```
label: { 
  "nostr/getPublicKey": "your translation",
}
```

see your first screenshot where it loads `badge.label.nostr/getPublicKey` (not `badge.label.nostr.getPublicKey`)

**Comment by manavdesai27** [comment]
> see your first screenshot where it loads `badge.label.nostr/getPublicKey` (not `badge.label.nostr.getPublicKey`)

Yes I know, it loads it that way. But I have changed the code such that `nostr/getPublicKey` is converted to `nostr.getPublicKey` before being sent for translation rendering. I have done so for the cleanliness of the translation file. What do you think about that @bumi .

**Comment by reneaaron** [comment]
Not sure if we want to maintain translated titles right now. 

What is the translation content of `nostr.getPublicKey` or `webln.addholdinvoice` for instance?

Another thing to consider here: 
Those badges would only show up if you give permanent permissions to those websites. You wouldn't see where you are using nostr if you don't do that. 

> I would like to see Nostr badge for websites I connected using Alby Nostr.

@bitcoinuser Why would you want to see that info on this list? What job does this badge help you to get done?

**Comment by manavdesai27** [comment]
@reneaaron What do you suggest instead then? Because I don't see a way to know where we are using nostr except the "permissions" way. One way could be adding a boolean flag, the same as `lnurlAuth`

**Comment by reneaaron** [comment]
@manavdesai27 Not sure, that's why I tagged @bitcoinuser to get a clear understanding about which problem we are actually trying to solve here.
